src/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+function Footer() {
+  const info = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
+  return (
+    <div id="footer">
+      <ul>
+        {info.map((el, i) => {
+          return <li key={i}>{el}</li>;
```

Comment:
Can you give more meaningful names instead of el on the map, please?

Reply:
Sure